src/main/resources/resources/logisim/strings/fpga/fpga.properties

```diff
@@ -2,6 +2,15 @@ FPGANotSupported = Not supported
 FPGASupported = Supported
 FPGAUnknown = Unknown
 #
+# file/FileWriter.java
+#
+fileCreateHDLFile = Creating HDL file : "%s"
```

Comment:
IIRC space in front  `:` is not correct in English (multiple). Also may I suggest rearranging the strings, the the file name is always the last element, so no matter how long the path there shown is it won't degrade message readablity:

```
fileHDLFileExists = HDL file already exists: %s
fileCreateScriptFile = Creating script file: %s
fileScriptsFileExists = Script file already exists: %s
fileUnableToCreate = Unable to create file: %s
fileUnableToWrite = Could not write to file: %s
```

Reply:
done